src/runtime/auto-prompt-manager-test.js

```diff
@@ -1909,20 +1909,16 @@ describes.realWin('AutoPromptManager', (env) => {
 
     it('should follow AudienceActionFlow without SecondPromptDelay experiment for Subscriptions', async () => {
       await autoPromptManager.showAutoPrompt({
-        autoPromptType: AutoPromptType.SUBSCRIPTION_LARGE,
+        autoPromptType: AutoPromptType.SUBSCRIPTION,
         alwaysShow: false,
         displayLargePromptFn: alternatePromptSpy,
       });
+      miniPromptApiMock.expects('create').once();
       await tick(7);
 
-      expect(startSpy).to.have.been.calledOnce;
-      expect(actionFlowSpy).to.have.been.calledWith(deps, {
-        action: 'TYPE_REWARDED_SURVEY',
-        onCancel: sandbox.match.any,
-        autoPromptType: AutoPromptType.SUBSCRIPTION_LARGE,
-      });
+      expect(startSpy).to.not.have.been.called;
       expect(alternatePromptSpy).to.not.have.been.called;
-      expect(autoPromptManager.getLastAudienceActionFlow()).to.not.equal(null);
+      expect(autoPromptManager.getLastAudienceActionFlow()).to.equal(null);
```

Comment:
This might be a dumb question, but why is switching to `AutoPromptType.SUBSCRIPTION` causing this change in behavior?

Reply:
This should have been `.to.equal(null)` originally, not sure why it wasn't failing before :)